cdci_data_analysis/analysis/instrument.py

```diff
@@ -334,6 +334,22 @@ def get_status_details(self,
             sentry.capture_message((f'ConnectionError detected when retrieving additional '
                                     f'information from a completed job '
                                     f'{ce}'))
+        except Unauthorized as ue:
+            detail_message = ""
```

Comment:
the job was submitted correctly so the error response should be different.

What seemed to have happened is that token has expired before the job completed. We should say this and that your result might have been partially or mostly ready, please resubmit the job. Also please consider to submit a job with longer token validity (i.e. login recently before the job submission)

Reply:
Also please add that if unclear they should contact us. Maybe even put URL to contact form?

Reply:
inserted the email address as well as a hint at using the "contact us" form on MModa